community/bolt/src/docs/dev/messaging.asciidoc

```diff
@@ -68,20 +69,20 @@ Once the client acknowledges the failure, it is then able to resend a corrected
 Protocol messages are represented as <<bolt-packstream-structures,serialized structures>>.
 
 [[bolt-message-structs-initialize]]
-==== INITIALIZE
+==== INIT
 
-The `INITIALIZE` message is a client message used once to initalize the session.
+The `INIT` message is a client message used once to initialize the session.
 This message is always the first message the client sends after <<bolt-handshake,negotiating protocol version>>.
-Sending any message other than `INITIALIZE` as the first message to the server will result in a `FAILURE` and the
+Sending any message other than `INIT` as the first message to the server will result in a `FAILURE` and the
```

Comment:
We could instead document the server-side state machine, which would cover this and other permitted/disallowed transitions.